fixtures/parser_tests.ts

```diff
@@ -184,24 +186,24 @@ function parserTests(parse: (file: string) => ParserResult) {
     it('finds test blocks within describe blocks', () => {
       const data = parse(`${fixtures}/dangerjs/travis-ci.example`);
       const descBlock = data.describeBlocks[1];
-      expect(descBlock.children.length).toBe(4);
+      expect(descBlock.children?.length).toBe(4);
```

Comment:
Can `children` really be not an (empty) array? I'm just wondering about the change between old and new code.